libraries/AP_Mount/AP_Mount_Params.cpp

```diff
@@ -173,6 +173,13 @@ const AP_Param::GroupInfo AP_Mount_Params::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("_OPTIONS", 16, AP_Mount_Params, options, 0),
 
+    // @Param: _RAT_TIMEOUT
+    // @DisplayName: Rate Request Timeout
```

Comment:
This is all looking good now thanks!

One small thing, could the Display Name be updated to include "Mount" at the front?  So "Mount rate request timeout" (or similar) would be nice.  The reason is just that without this it's not clear which subsystem is affected unless the user (or AI) looks at the parameter name.